sdk/keyvault/keyvault-admin/test/utils/common.ts

```diff
@@ -56,3 +58,31 @@ export function getSasToken() {
 
   return { blobStorageUri, blobSasToken };
 }
+
+/**
+ * The known API versions that we support.
+ */
+export const serviceVersions = ["7.2"] as const;
+
+/**
+ * Fetches the service version to test against. This version could be configured as part of CI
+ * and then passed through the environment in order to support testing prior service versions.
+ * @returns - The service version to test
+ */
+export function getServiceVersion(): SUPPORTED_API_VERSIONS {
+  return env.SERVICE_VERSION || LATEST_API_VERSION;
```

Comment:
This will always default to latest. The matrix key is `ServiceVersion` so that's what the env variable will be too. You'd have to change either the matrix key or this lookup.

Reply:
We have this bit here: https://github.com/Azure/azure-sdk-for-js/blob/main/eng/pipelines/templates/jobs/live.tests.yml#L136 (thanks for @maorleger for pointing this out to me)